### PR TITLE
Kitchen Sink

### PR DESCRIPTION
**Pull request by denricklaboradaglobe**
- Buttons
- Labels
- Progress Bars
- Text Inputs

NOTE: Navigation from `Home` to the designated showcase screens is not yet implemented as the React Native navigation library has not yet been imported.

### CODE REVIEW

File: example/src/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  scrollContentContainer: {
+    flexGrow: 1,
+    paddingHorizontal: 20,
+  },
```
Comment (tantan2395) [inline example/src/Home.tsx:33]: I think it's best if we add a ` justifyContent: 'center'` here?

File: example/src/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  scrollContentContainer: {
+    flexGrow: 1,
+    paddingHorizontal: 20,
```
Comment (tantan2395) [inline example/src/Home.tsx:32]: I think it's best if we add a ` justifyContent: 'center'` here?

Comment (tantan2395) [reply example/src/Home.tsx:32]: @denricklaboradaglobe

**Review by tantan2395 — CHANGES_REQUESTED** [review]

File: example/src/Home.tsx (hunk 2 above)
Comment (denricklaboradaglobe) [reply example/src/Home.tsx:32]: done

**Review by tantan2395 — APPROVED** [review]

**Pull request merged**